### PR TITLE
[SYCL] Fix the test/basic_tests/buffer/subbuffer.cpp

### PR DESCRIPTION
**Pull request by rbegam**
Signed-off-by: Rehana Begam <rehana.begam@intel.com>

### CODE REVIEW

File: sycl/test/basic_tests/buffer/subbuffer.cpp [hunk 1]
```diff
@@ -19,16 +19,19 @@
...
 #include <vector>
 
 void checkHostAccessor(cl::sycl::queue &q) {
-  std::size_t size =
-      q.get_device().get_info<cl::sycl::info::device::mem_base_addr_align>() /
-      8;
+  std::size_t subbuf_align =
+      q.get_device().get_info<cl::sycl::info::device::mem_base_addr_align>() / 8;
+  std::size_t size = subbuf_align;
+  size = std::max(size, 10 * 2 * sizeof(int)); // hold at least 20 elements
```
Comment (romanovvlad) [inline sycl/test/basic_tests/buffer/subbuffer.cpp]:
```suggestion
  std::size_t subbuf_align =
      q.get_device().get_info<cl::sycl::info::device::mem_base_addr_align>() / 8;
  std::size_t size = std::max(subbuf_align, 10 * 2 * sizeof(int)); // hold at least 20 elements
```

File: sycl/test/basic_tests/buffer/subbuffer.cpp [hunk 2]
```diff
@@ -19,16 +19,19 @@
 // 2) Exceptions if we trying to create sub buffer not according to spec
 
 #include <CL/sycl.hpp>
+#include <algorithm>
```
Comment (romanovvlad) [inline sycl/test/basic_tests/buffer/subbuffer.cpp]:
```suggestion
#include <CL/sycl.hpp>

#include <algorithm>
```

File: sycl/test/basic_tests/buffer/subbuffer.cpp [hunk 3]
```diff
@@ -59,9 +62,10 @@ void checkHostAccessor(cl::sycl::queue &q) {
 }
 
 void check1DSubBuffer(cl::sycl::queue &q) {
-  std::size_t size =
-      q.get_device().get_info<cl::sycl::info::device::mem_base_addr_align>() /
-      8;
+  std::size_t subbuf_align =
+      q.get_device().get_info<cl::sycl::info::device::mem_base_addr_align>() / 8;
+  std::size_t size = subbuf_align;
+  size = std::max(size, 32 * sizeof(int)); // hold at least 32 elements
```
Comment (romanovvlad) [inline sycl/test/basic_tests/buffer/subbuffer.cpp]:
```suggestion
  std::size_t subbuf_align =
      q.get_device().get_info<cl::sycl::info::device::mem_base_addr_align>() / 8;
   std::size_t size = std::max(subbuf_align, 32 * sizeof(int)); // hold at least 32 elements
```

**Comment by romanovvlad** [comment]
@rbegam Please, please apply clang format.

**Comment by rbegam** [comment]
@romanovvlad done. please review.

**Comment by rbegam** [comment]
@bader Merging is still blocked. there is no unresolved changes. Do you know how to enable merging?

**Comment by bader** [comment]
It is pending approval from @romanovvlad, who requested to make changes.

**Comment by rbegam** [comment]
thanks @bader and @romanovvlad for the approves.

**Pull request merged**